examples/react/src/workers/mupdf.worker.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference lib="webworker" />
+import * as Comlink from "comlink";
+import { ColorSpace, Document } from "mupdf";
+
+export const MUPDF_LOADED = "MUPDF_LOADED";
+
+const mupdfScript = import.meta.env.PROD
+  ? "/assets/mupdf.js"
+  : "/node_modules/mupdf/dist/mupdf.js";
+
+export class MupdfWorker {
+  private mupdf?: {
```

Comment:
Again, if going with higher level wrapper we might want to change this instance variable name to `mupdfjs` just for our own sanity.